boxes/templates/boxes/box_list.html

```diff
@@ -111,27 +111,28 @@ <h3 class="box__title text text-darkest">{{box.name}}</h3>
     <div class="row center-xs center-md center-lg">
       <div class="col-xs-12 col-sm-12 col-md-12">
         <h3 class="subtitle center-xs">Create New Box</h3>
-        <p class="smalltext text-lightest "> For others to be able to send you encrypted content through Hawkpost, you'll need to generate "boxes".</p> 
+        <p class="smalltext text-lightest "> For others to be able to send you encrypted content through Hawkpost, you'll need to generate "boxes".</p>
```

Comment:
Since this is a whitespace correction the classes list contains a trailing space as well.